app/controllers/contributors_controller.rb

```diff
@@ -58,6 +59,7 @@ def destroy
   end
 
   def count
+    # TODO: scope by organization?
```

Comment:
I moved this to the requests controller in the other PR since it is only used by the request form to display a message of how many contributors would receive the message with said tags.

```rb
def contributors_count
  render json: { count: @organization.contributors.with_tags(params[:tag_list]).count }
end
```

Reply:
I don't understand this. It's just adjusted with this PR https://github.com/tactilenews/100eyes/pull/1987 we can move it in a follow up

Reply:
Sure, no problem. It just doesn't make much sense to me that it ever lived in the contributors controller when it is only ever used by the requests controller.
Will you open a PR to move it? Or shall I? Do you think it should just stay where it is for some reason I haven't thought of?

Reply:
Feel free to move it wherever it makes sense. I've not yet looked into this.